### ISSUE TITLE
no way to tell the cavebot to move to an *EXACT* sqm

### ISSUE THREAD

**Issue by divinity76** [blackdtools/Blackd-Proxy-CLASSIC#102]
i need to move exactly to a specific SQM.. the solution should have been
```
:go1066626
move 106,662,6
IfTrue ($myx$#number<>#106) Goto $nlineoflabel:go1066626$
IfTrue ($myy$#number<>#662) Goto $nlineoflabel:go1066626$
```
- but with this code, if the bot manage to walk to the sqm minus 1x or minus 1y, it will just infinitely loop, without ever walking the last sqm. because the cavebot will stop once it considers my position to be "close enough", like 1 sqm away.. 

this is probably a good thing to avoid heuristic bot detection, meaning this is probably a *GOOD* thing for `antibanmode`, but with antibanmode disabled, this behavior just cause trouble.. 

i guess we could either implement a "moveExactSQM" command, portable across even antibanmode, or make the Move command more accurate when "antibanmode = 0" ...

**Comment by Neorisen** [comment]
Its about move 106,662,6 : P its not going to exact sqm because blackd implemented there some kind of "random walking" so you wont get detected that easily : P there's probably margin of 1 or 2 sqm so even if you're 1 or 2 sqm's away from target location, bot will still consider it as if he reached exact waypoint. Change it and bot will always go to that exact location

@edit
sorry I haven't read whole post ^^ anyway good luck with that

**Comment by divinity76** [comment]
@Neorisen yeah i think you're right - but this "random move" behavior should be confined to `antibanmode != 0` - but what do you think about PR #103 ?

**Comment by Neorisen** [comment]
I haven't been playing with blackd code for a long time, but it looks good : P you have to be sure that antibanmode doesnt affect something else though, because that might get you banned(if u care about that ofc)

**Comment by divinity76** [comment]
@Neorisen i only play on OT servers, and i always play with `antibanmode = 0` ^^  and in fact, i think BlackD Proxy isn't compatible with Tibia 11.x anyway (i believe BattlEye and the tibia client QT-rewrite broke BlackD Proxy, and @blackdtools doesn't have time/motivation to fix it, as the QT-rewrite and BattlEye is a MAJOR challenge)

**Comment by Neorisen** [comment]
@divinity76 Yeah im aware of that but some OT servers are still banning bots : D You made me even look whats going on in otservlist and tibia.com :P Tibia is still alive in 2019 which is pretty amazing
